src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -668,3 +696,11 @@ private class FastqDemultiplexer(val sampleInfos: Seq[SampleInfo],
     DemuxResult(sampleInfo=sampleInfo, numMismatches=numMismatches, records=records)
   }
 }
+
+sealed trait OutputType extends EnumEntry
+object OutputType extends FgBioEnum[OutputType] {
```

Comment:
I would suggest `Bam`, `Fastq` and `BamAndFastq` for the values for the enum. Or ... actually thinking about it more another option would be to just have `Bam` and `Fastq` and then let the user specify `--output-type Bam Fastq` to get both.  Just a thought.  I think that's cuter, but perhaps less obvious to users.